public/404.html

```diff
@@ -9,24 +9,18 @@
 </head>
 <body>
 
-     <div class="nav_bar">
-          <img id="quiz_icon" src="https://static.thenounproject.com/png/28038-200.png"/>
-
-
-          <div id="nav_search">
-               <input type="text" id="nav_search_box" placeholder="     Search"/>
-               <button type="button" id="nav_search_button"><i class="fas fa-search"></i></button>
-          </div>
-
-          <button type="button" id="nav_menu_button"> + Create Quiz </button>
-     </div>
-
-     <p class="4041"> Oops! </p>
-
-     <p class="4042"> We couldn't find what you were looking for. </p>
-
+  <div class="nav_bar">
+       <img id="quiz_icon" src="https://static.thenounproject.com/png/28038-200.png"/>
 
+       <div id="nav_search">
+            <input type="text" id="nav_search_box" placeholder="     Search"/>
+           <button type="button" id="nav_search_button"><i class="fas fa-search"></i></button>
+       </div>
 
+       <button type="button" id="nav_menu_button"> + Create Quiz </button>
+  </div>
 
+     <p class="error_oops"> Oops! </p>
+     <p class="error_finding"> We couldn't find what you were looking for. </p>
```

Comment:
These names are better, but it would be best to name elements in such a way that they still make sense even if the content changes. If we changed the message to say `This is awkward` instead of `Oops` we would have to rename all these `error_oops`s to `error_ awkward `s and that doesn't really make good sense. Could you pick a name that describes what this text *is* and not what it *says*?